### PR TITLE
Alerts: Restructured into smaller and more concise sections

### PR DESCRIPTION
**Pull request by adnanrahic**
I've broken up the long pages into smaller pages. This layout is more concise and we re-use parts for different alert types, instead of writing dedicated sections for all alert types.
_Note to self: I need to update the alerts quick links in SC before merging this._

### CODE REVIEW

File: docs/alerts/account-default-hooks.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Besides *account-default notification hooks*, you can specify additional hooks that pertain only to this particular alert rule. The following alert would send notifications to 4 different destinations when triggered.
+
+![image alt text](../images/alerts/image_14.png)
+
+Each alert gives you the option to opt-out from **"Using account-default hooks for that alert"**, by simply turning off that toggle.
+
+![image alt text](../images/alerts/image_15.png)
+
+If you turn off notifications for an *alert rule* the only consequence of that alert being triggered will be to generate events in the [Events view](../events).
```
Comment (mbonaci) [inline docs/alerts/account-default-hooks.md]: While you're at it, maybe better to point this to "our" `Alert Events` page (which points to Events)

File: docs/alerts/alert-notifications.md [hunk 2]
```diff
@@ -1,65 +1,5 @@
...
-
-![image alt text](../images/alerts/image_7.png)
-
- When you tag an alert with a color it looks like this:
-
-![image alt text](../images/alerts/image_8.png)
-
-### Notifications tab
+title: Alerts Notifications
+description: Configuring notirications for Alerts
```
Comment (mbonaci) [inline docs/alerts/alert-notifications.md]: notirications

File: docs/alerts/alerts-meta.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+title: Alerts Meta and Tags
+description: Configuring metadata for Alerts
+
+Meta tab allows you to tag alert rule with a specific color and specify further details, like Runbook and Description.
```
Comment (otisg) [inline docs/alerts/alerts-meta.md]: Meta tab lets you

File: docs/alerts/creating-logs-alerts.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+title: Creating Logs Alerts
+description: Step-by-step alert creation instructions for Logs
+
+In a Logs app, *saved queries* are used to save search queries that you want to reuse.
```
Comment (otisg) [inline docs/alerts/creating-logs-alerts.md]: app => App

File: docs/alerts/creating-metrics-alerts.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+![image alt text](../images/alerts/metric-alert.png)
+
+If you don't want to use a specific threshold value and instead get notified when the CPU usage deviates from a continuously computed baseline, you'd change `Alert type` to `Anomaly alert`.
+
+![image alt text](../images/alerts/metric-anomaly-alert.png)
+
+Notice that the chart changed to help you get a sense of what would constitute an anomalous value. In this case, the dots outside the gray confidence interval.
+
+The confidence interval is an approximation of Sematext Cloud's anomaly detection algorithm, so don't expect each and every red dot on the chart to have triggered the alert.
```
Comment (otisg) [inline docs/alerts/creating-metrics-alerts.md]: The confidence interval is an approximation ==> The confidence interval shown in the preview is an approximation

**Pull request merged**